.golangci.yml

```diff
@@ -11,7 +11,6 @@ linters:
     - exportloopref
     - gocritic
     - godot
-    - golint
```

Comment:
LGTM. Maybe still remain it, though it's deprecated.

Reply:
i think no need to keep `golint`, becasue it's check have been replaced by `revive` (we already have this linter)